otter/test/convergence/test_service.py

```diff
@@ -330,13 +359,13 @@ def test_unexpected_errors(self):
         dirty flag.
         """
         expected_error = RuntimeError('oh no!')
-        sequence = SequenceDispatcher([
+        sequence = [
```

Comment:
For the record, I actually decided to go a different way with this -- I mean, I still updated the test to be more explicit, but I decided to add the group to "recently converged" even in the case of error, so that even when a group is messing up it won't be converged too quickly again.